test/src/tests-consent.ts

```diff
@@ -710,6 +710,7 @@ describe('Consent', function() {
         done();
     });
 
+    // TODO: Can we remove this now?
```

Comment:
Technically it's still in the API and we never gave notice.  Let's flag as a TODO: Remove in next major version.  As part of modernization.